1DIP/deriv2_cgls.ipynb

```diff
@@ -1,11 +1,36 @@
 {
  "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "602dbdfe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# -*- coding: utf-8 -*-\n",
+    "#  Copyright 2023 United Kingdom Research and Innovation\n",
+    "#\n",
+    "#  Licensed under the Apache License, Version 2.0 (the \"License\");\n",
+    "#  you may not use this file except in compliance with the License.\n",
+    "#  You may obtain a copy of the License at\n",
+    "#\n",
+    "#      http://www.apache.org/licenses/LICENSE-2.0\n",
+    "#\n",
+    "#  Unless required by applicable law or agreed to in writing, software\n",
+    "#  distributed under the License is distributed on an \"AS IS\" BASIS,\n",
+    "#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n",
+    "#  See the License for the specific language governing permissions and\n",
+    "#  limitations under the License.\n",
+    "#\n",
+    "#   Authored by:  Bill Lionheart (University of Manchester), Margaret DUff (STFC - UKRI)"
```

Comment:
I suggest we add a new line "Reviewed by: " or "Reviewed/edited by:" or similar and add names of those of us who reviewed/edited to keep the user contributor as the author.

Reply:
Sounds good - will update this